framework/source/class/qx/data/marshal/Json.js

```diff
@@ -301,7 +317,15 @@ qx.Class.define("qx.data.marshal.Json",
       } else {
         var className = "qx.data.model." + hash;
         var clazz = qx.Class.getByName(className);
-        if (!clazz) {
+        if (!clazz)
+        {
+          // Extra check for possible bubble-event feature inconsistency
+          var className_ = className.replace(/~/g, '"');
```

Comment:
Please can you use a different name for the variable - the underscore does not help describe it's purpose, eg `bubbleClassName` ?